### PR TITLE
DOC: add documentation for constants

### PR DESCRIPTION
**Pull request by guoci**

### CODE REVIEW

File: numpy/doc/constants.py [hunk 1]
```diff
@@ -363,6 +365,26 @@ def add_newdoc(module, name, doc):
...
+
+    References
+    ----------
+    https://en.wikipedia.org/wiki/Pi
+
+    """)
+
+add_newdoc('numpy', 'euler_gamma',
+    """
+    ``γ = 0.5772156649015328606065120900824024310421...``
```
Comment (charris) [inline numpy/doc/constants.py]: This doesn't work for python and standard "ascii" encoding.

**Comment by charris** [comment]
Now we have to figure out if this works when the docs are built. Have you tried that?

**Comment by charris** [comment]
@stephan Does numpydoc/sphinx work with utf-8 docstrings?

**Comment by guoci** [comment]
@charris  Yes, I have built the docs on my machine.

**Comment by charris** [comment]
OK, let's give it a shot. Thanks @guoci

**Pull request merged**